index.js

```diff
@@ -436,6 +439,14 @@ function parse (args, opts) {
     })
   }
 
+  // set all config objects passed in opts
+  function setConfigObjects () {
+    if (typeof configObjects === 'undefined') return
```

Comment:
The check for `undefined` seems to be handled by line 22: `opts.configObjects || []`.

Perhaps it would be better to make sure `configObjects` is an Array here:

```js
if (!Array.isArray(configObjects)) configObjects = [].concat(configObjects)
```